runtime/server/batch_query.go

```diff
@@ -0,0 +1,177 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	"github.com/rilldata/rill/runtime/server/auth"
+)
+
+func (s *Server) QueryBatch(req *runtimev1.QueryBatchRequest, srv runtimev1.QueryService_QueryBatchServer) error {
+	ctx := srv.Context()
+
+	if !auth.GetClaims(ctx).CanInstance(req.InstanceId, auth.ReadOLAP) {
+		return ErrForbidden
+	}
+	var wg sync.WaitGroup
+
+	for _, query := range req.Queries {
+		wg.Add(1)
+		go func(query *runtimev1.QueryBatchSingleRequest) {
+			defer wg.Done()
+
+			resp := s.forwardQuery(ctx, query)
+
+			if err := srv.Send(resp); err != nil {
+				s.logger.Debug(fmt.Sprintf("Profiling Query Error: %v", err))
+			}
```

Comment:
Are there any cases where `Send` will error where we shouldn't stop processing queries and terminate the batch? If not, consider using `errgroup` instead